docs/pages/User/GettingStarted/demo_k8s_on_desktop.md

```diff
@@ -0,0 +1,216 @@
+  * a `pod` executing the VNC viewer, started in the local machine (i.e., on the same node where you run `cloudify`).
+  
+## Installation of the required software
+To install all the required software we need to follow this steps:
+
+1. [Install k3s](#install-k3s) in both clusters;
+2. [Install liqo](#install-liqo) in both clusters; 
+3. [Install KubernetesOnDesktop](#install-kubernetesondesktop) in the *local cluster*. Note that the foreign cluster simply runs a vanilla Liqo, without any other software.
+
+### Install k3s
+Assuming you already have two Linux machines (or Virtual Machines) up and running in the same LAN, we can install [k3s](https://k3s.io/) by using the official script as documented in the [K3s Quick-Start Guide](https://rancher.com/docs/k3s/latest/en/quick-start/).
+So, you only need to run the following command:
+
+```bash
+curl -sfL https://get.k3s.io | sh -
+```
+
+A tiny customization of the above default install is required for [liqo](https://liqo.io) to work.
+When the script ends, you need to modify the `/etc/systemd/system/k3s.service` file by adding the `--kube-apiserver-arg anonymous-auth=true` service execution parameter as in the following command:
+
+```bash
+sudo sed -i "s#server#server --kube-apiserver-arg anonymous-auth=true#" /etc/systemd/system/k3s.service
+```
+
+{{%expand "After this operation, your `k3s.service` file should look like the following:" %}}
+
+```
+[Unit]
+Description=Lightweight Kubernetes
+Documentation=https://k3s.io
+Wants=network-online.target
+
+[Install]
+WantedBy=multi-user.target
+
+[Service]
+Type=notify
+EnvironmentFile=/etc/systemd/system/k3s.service.env
+KillMode=process
+Delegate=yes
+# Having non-zero Limit*s causes performance problems due to accounting overhead
+# in the kernel. We recommend using cgroups to do container-local accounting.
+LimitNOFILE=1048576
+LimitNPROC=infinity
+LimitCORE=infinity
+TasksMax=infinity
+TimeoutStartSec=0
+Restart=always
+RestartSec=5s
+ExecStartPre=-/sbin/modprobe br_netfilter
+ExecStartPre=-/sbin/modprobe overlay
+ExecStart=/usr/local/bin/k3s \
+    server --kube-apiserver-arg anonymous-auth=true \
+
+```
+{{% /expand%}}
+
+{{%expand "**Note**: If you want to exploit any available NVIDIA GPUs in the *foreign cluster* you also have to do in that cluster as follows:" %}}
+1. [Install in the required NVIDIA CUDA driver](https://github.com/NVIDIA/nvidia-docker/wiki/Frequently-Asked-Questions#how-do-i-install-the-nvidia-driver);
+2. [Install the Docker engine](https://docs.docker.com/engine/install/);
+3. [Install the `nvidia-container-runtime`](https://github.com/nvidia/nvidia-container-runtime#installation);
+4. Add the `--docker` service execution parameter in the `/etc/systemd/system/k3s.service` file by executing the following command:
```

Comment:
Add the `--docker` service execution parameter ==> Add the `--docker` service execution parameter TO DO WHAT?
Tell people not only what to do, but also why is this needed (briefly, very briefly)

Reply:
Fixed